python/lib/dcoscli/dcoscli/marathon/main.py

```diff
@@ -29,6 +29,7 @@ def main(argv):
 
 
 @decorate_docopt_usage
+@cluster_version_check
```

Comment:
Would we put this decorator on top of all commands? Are there cases where it wouldn't be called?

Reply:
I had thought we wouldn't put them on every command, just the ones that are version specific so the ones the wrapper doesn't have. But maybe we want it for all commands to have the warning all over on the CLI people use for older clusters.

Reply:
gotcha, sounds good, it gives us the flexibility to choose what we want to do :+1: